src/components/Book/CategoryView.tsx

```diff
@@ -1,4 +1,6 @@
 import classNames from 'classnames';
+import React from 'react';
+import { useMediaQuery } from 'react-responsive';
```

Comment:
Why is this package needed?
If there is something you need to do in different break points, it should be done in CSS not as adding another dependency.